package.json

```diff
@@ -78,6 +78,7 @@
     "carbon-icons": "7.0.7",
     "classnames": "^2.2.5",
     "core-js": "2.6.5",
+    "deepdash": "^4.4.2",
```

Comment:
since we're only using deepdash in the storybook, it should be in devDependencies, not dependencies

Reply:
Moved to devDeps